module/attribute/src/Application/Validator/AttributeCode.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Attribute\Application\Validator;
+
+use Ergonode\Attribute\Domain\ValueObject\AttributeCode as ValueObject;
+use Ergonode\SharedKernel\Application\Validator\SystemCode;
+
+/**
+ * @Annotation
+ */
+class AttributeCode extends SystemCode
```

Comment:
```suggestion
class AttributeCodeCorrect extends SystemCode
```
Because in my opinion, it could be confused with `Ergonode\Attribute\Domain\ValueObject\AttributeCode`

Reply:
I'd avoid similar exceptions - this most probably won't be the only place where there will be the same class name for model of some sort and its constraint